index.html

```diff
@@ -19,25 +19,30 @@
     <main class="main-container">
       <section class="header-mobile">
         <!-- visible to mobile platforms only-->
-        <div><a href="#">Welcome</a></div>
-        <div><img src="images/toggle.png" alt="toggle menu icon" /></div>
+        <div><a href="#" class="logo-brand">Welcome</a></div>
+        <div class="toggle-icon">
+          <img src="images/toggle.png" alt="toggle menu icon" />
+        </div>
+        <div class="close-icon hide">
+          <img src="./images/Cancel.svg" alt="" class="close-icon" />
+        </div>
         <!-- end visible to mobile only-->
       </section>
```

Comment:
- [ ] To make our design exactly the same as that of Figma, let's move this section a little bit towards the top.